src/java/picard/vcf/GenotypeConcordanceCounts.java

```diff
@@ -272,19 +272,19 @@ public int getSum(final Set<TruthState> truthStateSet, final Set<CallState> call
     /**
      * Returns the sum of all pairs of tuples defined by the truth state set and call state set.
      */
-    public int getSum() {
+    public long getSum() {
         return getSum(new HashSet<TruthState>(Arrays.asList(TruthState.values())), new HashSet<CallState>(Arrays.asList(CallState.values())));
     }
 
     /**
      * Returns the total number of times each contingency state is encountered, summed across all truth/call state pairs.
      */
-    public Map<ContingencyState, Integer> getContingencyStateCounts(final GenotypeConcordanceScheme scheme) {
+    public Map<ContingencyState, Long> getContingencyStateCounts(final GenotypeConcordanceScheme scheme) {
         scheme.validateScheme();
 
-        final Map<ContingencyState, Integer> counts = new HashMap<ContingencyState, Integer>();
+        final Map<ContingencyState, Long> counts = new HashMap<ContingencyState, Long>();
         for (final ContingencyState contingencyState : ContingencyState.values()) {
-            counts.put(contingencyState, 0);
+            counts.put(contingencyState, (long)0);
```

Comment:
Haha I worked at McGill University Genome center and I did send pull
requests to picard a few times.

I thought this was about a pull request I had sent awhile ago.

In anycase, still good form to use literals.

Louis

On Tue, Sep 29, 2015 at 12:12 PM, Louis Bergelson <notifications@github.com>
wrote:

> In src/java/picard/vcf/GenotypeConcordanceCounts.java
> <https://github.com/broadinstitute/picard/pull/296#discussion_r40692630>:
>
> >          for (final ContingencyState contingencyState : ContingencyState.values()) {
> > -            counts.put(contingencyState, 0);
> > +            counts.put(contingencyState, (long)0);
>
> Who's this alternate Louis? There can be only 1!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/picard/pull/296/files#r40692630>.
>